index.html

```diff
@@ -887,9 +887,10 @@ <h2>
             The <a>show()</a> method is called when a developer wants to begin
             user interaction for the payment request. The <a>show()</a> method
             returns a <a>Promise</a> that will be resolved when the <a>user
-            accepts the payment request</a>. Some kind of user interface will
-            be presented to the user to facilitate the payment request after
-            the <a>show()</a> method returns.
+            accepts the payment request</a> or rejected if the <a>user aborts
+            the payment request</a>. Some kind of user interface will be
```

Comment:
There's a lot of failure conditions, and listing only one of them might be a bit more confusing than helpful.

Reply:
Agree. Reverting.